--- id3ed-1.10.3/Makefile.in~	Thu Sep 20 06:45:11 2001
+++ id3ed-1.10.3/Makefile.in	Thu Sep 20 06:45:59 2001
@@ -14,6 +14,7 @@
 bindir=@bindir@
 mandir=@mandir@
 install=@INSTALL@
+LDFLAGS=@LDFLAGS@
 
 DESTDIR=
 
@@ -22,13 +23,14 @@
 OBJS=id3ed.o misc.o
 
 id3ed: $(OBJS) .depend
-	$(CXX) $(CXXFLAGS) $(OBJS) -o $@ $(LIBS)
+	$(CXX) $(CXXFLAGS) $(LDFLAGS) $(OBJS) -o $@ $(LIBS)
 
 dep depend .depend:
 	$(CXX) -MM *.cc > .depend
 
 install: all
-	$(install) -s -m 0755 id3ed $(DESTDIR)$(bindir)
+	$(install) -d $(DESTDIR)$(bindir) $(DESTDIR)$(mandir)/man1
+	$(install) -m 0755 id3ed $(DESTDIR)$(bindir)
 	$(install) -m 0644 id3ed.1 $(DESTDIR)$(mandir)/man1
 
 uninstall:
