diff -Nru id3ed-1.10.1/Makefile.in id3ed-1.10.1.new/Makefile.in
--- id3ed-1.10.1/Makefile.in	Thu Sep  7 06:09:33 2000
+++ id3ed-1.10.1.new/Makefile.in	Thu Sep  7 21:35:46 2000
@@ -14,6 +14,7 @@
 bindir=@bindir@
 mandir=@mandir@
 install=@INSTALL@
+LDFLAGS=@LDFLAGS@
 
 DESTDIR=
 
@@ -22,14 +23,15 @@
 OBJS=id3ed.o misc.o
 
 id3ed: $(OBJS) .depend
-	$(CXX) $(CXXFLAGS) $(OBJS) -o $@ $(LIBS)
+	$(CXX) $(CXXFLAGS) $(LDFLAGS) $(OBJS) -o $@ $(LIBS)
 
 dep depend .depend:
 	$(CXX) -MM *.cc > .depend
 
 install: all
-	$(install) -s -o root -g bin -m 0755 id3ed $(DESTDIR)$(bindir)
-	$(install) -o root -g bin -m 0644 id3ed.1 $(DESTDIR)$(mandir)/man1
+	$(install) -d $(DESTDIR)$(bindir) $(DESTDIR)$(mandir)/man1
+	$(install) id3ed $(DESTDIR)$(bindir)
+	$(install) -m 0644 id3ed.1 $(DESTDIR)$(mandir)/man1
 
 uninstall:
 	-rm $(DESTDIR)$(bindir)/id3ed
